pom.xml

```diff
@@ -1557,12 +1508,10 @@
                                 <path>
                                     <groupId>${project.groupId}</groupId>
                                     <artifactId>error-prone-contrib</artifactId>
-                                    <version>${project.version}</version>
                                 </path>
                                 <path>
                                     <groupId>${project.groupId}</groupId>
                                     <artifactId>refaster-runner</artifactId>
-                                    <version>${project.version}</version>
                                 </path>
                             </annotationProcessorPaths>
                         </configuration>
```

Comment:
IIUC we can now drop this one: https://github.com/PicnicSupermarket/error-prone-support/pull/515/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R1596, right?

Reply:
Hmm, that one counteracts [the opposite config on line 911](https://github.com/PicnicSupermarket/error-prone-support/pull/515/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R911). This way warnings are disabled if `-Dverification.skip` is specified.